README.md

```diff
@@ -1,59 +1,219 @@
-[npm-image]:https://img.shields.io/npm/v/@moxy/{package-name}.svg
-[build-status-url]:https://github.com/moxystudio/{package-name}/actions
-[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/{package-name}/Node%20CI/master
-[codecov-url]:https://codecov.io/gh/moxystudio/{package-name}
-[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/{package-name}/master.svg
-[david-dm-url]:https://david-dm.org/moxystudio/{package-name}
-[david-dm-image]:https://img.shields.io/david/moxystudio/{package-name}.svg
-[david-dm-dev-url]:https://david-dm.org/moxystudio/{package-name}?type=dev
-[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/{package-name}.svg
+[npm-url]:https://npmjs.org/package/@moxy/react-contentful-image
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-contentful-image.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-contentful-image.svg
+[build-status-url]:https://github.com/moxystudio/react-contentful-image/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-contentful-image/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-contentful-image
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-contentful-image/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-contentful-image
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-contentful-image.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-contentful-image?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-contentful-image.svg
 
-{package-description}
+A react image renderer that uses the Contentful Images API.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-contentful-image
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+[Contentful](https://www.contentful.com/) provides a very powerful [Images API](https://www.contentful.com/developers/docs/references/images-api/) that besides retrieving image files, provides manipulation features such as resizing, cropping and compressing.
+
+This react component returns a `<picture>` element. If no manipulations are made, the `<picture>` contains at least one `<source>` element, otherwise it will have two `<source>`. It also contains the native `<img>` as fallback for browsers that do not support the `<picture>` element.
 
 ## Usage
 
-{package-usage-example}
+```js
+import React from 'react';
+import ContentfulImage from '@moxy/react-contentful-image';
+
+const src = "//images.ctfassets.net/yadj1kx9rmg0/wtrHxeu3zEoEce2MokCSi/cf6f68efdcf625fdc060607df0f3baef/quwowooybuqbl6ntboz3.jpg";
+
+const MyComponent = (props) => (
+    <div {...props}>
+        <ContentfulImage
+            src={ src }
```

Comment:
It seems that we can only pass the url as a string right?

What if this component actually supported this, but also supported passing an image asset data structure from contentful?

This way we wouldn't have to go for `fields.file.url` every time.

So, if we passed a string, we assumed it as an url. If we passed an object, the component would try to fetch the url by itself.